docs/relay.md

```diff
@@ -143,9 +143,9 @@ const userType = new GraphQLObjectType({
       type: GraphQLString
     },
     tasks: {
-      type: userTaskConnection.connectionType,
-      args: userTaskConnection.connectionArgs,
-      resolve: userTaskConnection.resolve
+      type: userTaskConnection.field.type,
+      args: userTaskConnection.field.args,
+      resolve: userTaskConnection.field.resolve,
```

Comment:
i meant more something like `/* {type, args, resolve} * /`

Reply:
as a comment next to `tasks: userTaskConnection.field` you mean?

Reply:
yeah just document the structure, leave as much documentation as is otherwise and just add an extra example